src/index.html

```diff
@@ -1,12 +1,285 @@
+<!-- #region head -->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a href="#" class="img"><img src="/images/logo.png" alt="Moyo_logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="nav__link is-active">Apple</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a href="#"
+              class="nav__link"
+              data-qa="nav-hover">
+              Laptops & Computers
+            </a>
```

Comment:
Need to fix code style everywhere
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
```